constraint_utils/generic.py

```diff
@@ -71,8 +89,20 @@ def count_variants(ht: hl.Table,
                    omit_methylation: bool = False, return_type_only: bool = False,
                    force_grouping: bool = False, singleton_expression: hl.expr.BooleanExpression = None,
                    impose_high_af_cutoff_here: bool = False) -> Union[hl.Table, Any]:
-    """
-    Count variants by context, ref, alt, methylation_level
+    """Count number of observed variants by context, ref, alt, methylation_level
+
+    :param ht: Input Hail table.
+    :param count_singletons: Whether to count singletons. Defaults to False.
+    :param count_downsamplings: List of populations to use for downsampling counts. Defaults to ().
+    :param additional_grouping: Additional features to group by. i.e. exome_coverage.
```

Comment:
This is the only one where you don't define what the default is

Reply:
The code didn't define the default, so I skipped it.

Reply:
Code is `additional_grouping: Optional[List[str]] = ()`